rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+[Iterating with Oversight]: #iterating-with-oversight
+
+Roadmaps MUST be approved by Oversight rather than DCP PM which is a minor variation from the current RFC Approval process for Governance RFC(s).
+
+The Roadmap Shepherd MUST send the draft Roadmap to Oversight a week in advance of a review meeting between the Project Leads and Oversight.
+
+The Project Leads MUST iterate with Oversight to review and revise the draft Roadmap until it meets with Oversight approval.
+
+Once approved, the Roadmap Shepherd will follow the normal RFC process for [approved RFC(s)](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md#approving-rfcs).
+
+### Updating the Roadmap
+
+The Roadmap MAY be updated with Project Lead consensus and Oversight approval for a number of reasons including but not limited to:
+
+* A Theme is to be added or deleted.
+* Additions or revision to Objectives change the original intent of the Theme.
+* Emerging conditions require substantive changes to the original Roadmap direction.
+
+Requests to update the Roadmap MUST be forwarded to the Roadmap Shepherd with a detailed description of the proposed changes, including the rationale for the updates, a timeline, and priority.
+
+The Roadmap Shepherd MUST add the request to the next Project Leads meeting agenda for review.
+
+The Roadmap Shepherd and Project Leads MUST review the update request and determine if there is rough consensus.
+
+If consensus is reached, the Roadmap Shepherd MUST refresh the Roadmap and send to Oversight. 
+
+Oversight MUST review the changes and approve.
+
+The Roadmap Shepherd MUST update the previously published Roadmap RFC in the *dcp-community* repo and announce its approval in a similar fashion to the original RFC approval.
+
+## Detailed Design for Planning and Execution
+
+### ZenHub for Project Management
+
+The [ZenHub](https://www.zenhub.com/) project management application has been partially adopted in the DCP community. This RFC fully commits the community to ZenHub as our *one source of truth* for project management and engineering plans.
+
+DCP SHOULD NOT duplicate the ZenHub DCP Board in other documents or spreadsheets. The Board and its reports represent the definitive list of DCP work items for a Release and their current state. 
+
+### DCP Product Backlog
+
+The DCP ZenHub board is the Product Backlog for DCP. All DCP component repos that will assign items to a DCP Release and Milestone MUST be added to the ZenHub *DCP Backlogs* workspace. The [DCP PM Team](https://github.com/orgs/HumanCellAtlas/teams/dcp-pm-team) in the Human Cell Atlas GitHub organization MUST have write access to all the repo(s) in the *DCP workspace*.
+
+The top-level *dcp* repo MUST contain **all** significant work that is important to the DCP community as a whole and require visibility and prioritization. This includes Roadmap objectives and product-wide technical requirements (e.g. “components should have logs...“). 
```

Comment:
What does "DCP community as a whole" mean?

Reply:
Good point. I believe that the original author wanted to emphasize that work might be focused in a specific component but we would still want it visible on the dcp-level board. But- will update to _DCP community ~as a whole~_ since it seems redundant.

Reply:
Addressed by https://github.com/HumanCellAtlas/dcp-community/pull/92/commits/8eb925ffb6beaab53e0319ff05bd5f01ed5c52e9